src/common/utils/fetchExportDataByIds.js

```diff
@@ -0,0 +1,15 @@
+import { chunk } from 'lodash';
+
+import { batchFetch } from '@folio/stripes-acq-components';
+
+export const fetchExportDataByIds = async (mutator, ids, query) => {
+  const batchedIds = chunk(ids, 50);
```

Comment:
since `batchFetch` could fail on parallel loading of many chunks maybe it makes sense to put code right there. Isn't this specific to order lines?